### DISCUSSION TITLE
posts/%EB%A9%94%EC%84%B8%EC%A7%80-%EC%9E%AC%EC%A0%84%EC%86%A1-%ED%8A%B8%EB%9F%AC%EB%B8%94%EC%8A%88%ED%8C%85%EA%B8%B0/

### DISCUSSION THREAD

**Discussion by giscus** [joonamin/joonamin.github.io#9] [Comments]
# posts/%EB%A9%94%EC%84%B8%EC%A7%80-%EC%9E%AC%EC%A0%84%EC%86%A1-%ED%8A%B8%EB%9F%AC%EB%B8%94%EC%8A%88%ED%8C%85%EA%B8%B0/

서론

https://joonamin.github.io/posts/%EB%A9%94%EC%84%B8%EC%A7%80-%EC%9E%AC%EC%A0%84%EC%86%A1-%ED%8A%B8%EB%9F%AC%EB%B8%94%EC%8A%88%ED%8C%85%EA%B8%B0/

**Comment by baesunyoung6767** [comment]
좋은 글 감사합니다~^^* 🥰

**Reply by joonamin** [reply]
항상 감사합니다 :)
